common/recorderfile.cpp

```diff
@@ -130,12 +132,20 @@ bool PlaybackFile::parseHeader() {
 }
 
 bool PlaybackFile::checkPlaybackFileVersion() {
-	uint32 version;
-	version = _readStream->readUint32LE();
-	if (version != RECORD_VERSION) {
-		warning("Incorrect playback file version. Expected version %d, but got %d.", RECORD_VERSION, version);
+	_version = _readStream->readUint32LE();
+	switch (_version) {
+	case 2:
+		break;
+	case 1:
+		_trackScreenUpdate = false;
+		break;
+	default:
+		warning("Unknown playback file version %d. Maximum supported version is %d.", _version, RECORD_VERSION);
 		return false;
 	}
+	if (_version != RECORD_VERSION) {
```

Comment:
Last time modified the event recorder @sev-  said that bumping the version of the recorded file isn't needed, as there basically aren't any wild living recorded files around yet. I suppose this is still true.

Reply:
No worries, I've removed the backwards-compatibility code. On the basis that it's okay for older files not to work, I've taken the opportunity to switch the file format to using big-endian everywhere, mostly so that external tooling doesn't need to make a special parsing exception for the screenshot THMB block.